packages/dd-trace/test/config.spec.js

```diff
@@ -270,21 +269,32 @@ describe('Config', () => {
       { name: 'profiling.sourceMap', value: true, origin: 'default' },
       { name: 'profiling.exporters', value: 'agent', origin: 'default' },
       { name: 'startupLogs', value: false, origin: 'default' },
-      { name: 'telemetry.enabled', value: true, origin: 'default' },
+      { name: 'telemetry.enabled', value: true, origin: 'env_var' },
       { name: 'telemetry.heartbeatInterval', value: 60000, origin: 'default' },
       { name: 'telemetry.debug', value: false, origin: 'default' },
-      { name: 'telemetry.metrics', value: false, origin: 'default' },
+      { name: 'telemetry.metrics', value: true, origin: 'default' },
       { name: 'telemetry.dependencyCollection', value: true, origin: 'default' },
+      { name: 'telemetry.logCollection', value: false, origin: 'default' },
       { name: 'tagsHeaderMaxLength', value: 512, origin: 'default' },
       { name: 'appsec.enabled', value: undefined, origin: 'default' },
       { name: 'appsec.rules', value: undefined, origin: 'default' },
       { name: 'appsec.rateLimit', value: 100, origin: 'default' },
       { name: 'appsec.wafTimeout', value: 5e3, origin: 'default' },
       { name: 'appsec.blockedTemplateHtml', value: undefined, origin: 'default' },
       { name: 'appsec.blockedTemplateJson', value: undefined, origin: 'default' },
-      { name: 'appsec.eventTracking.enabled', value: true, origin: 'default' },
-      { name: 'appsec.eventTracking.mode', value: 'safe', origin: 'default' },
-      { name: 'remoteConfig.enabled', value: true, origin: 'default' },
+      {
+        name: 'appsec.obfuscatorKeyRegex',
+        // eslint-disable-next-line max-len
+        value: '(?i)(?:p(?:ass)?w(?:or)?d|pass(?:_?phrase)?|secret|(?:api_?|private_?|public_?)key)|token|consumer_?(?:id|key|secret)|sign(?:ed|ature)|bearer|authorization',
+        origin: 'default'
+      },
+      {
+        name: 'appsec.obfuscatorValueRegex',
+        // eslint-disable-next-line max-len
+        value: '(?i)(?:p(?:ass)?w(?:or)?d|pass(?:_?phrase)?|secret|(?:api_?|private_?|public_?|access_?|secret_?)key(?:_?id)?|token|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)(?:\\s*=[^;]|"\\s*:\\s*"[^"]+")|bearer\\s+[a-z0-9\\._\\-]+|token:[a-z0-9]{13}|gh[opsu]_[0-9a-zA-Z]{36}|ey[I-L][\\w=-]+\\.ey[I-L][\\w=-]+(?:\\.[\\w.+\\/=-]+)?|[\\-]{5}BEGIN[a-z\\s]+PRIVATE\\sKEY[\\-]{5}[^\\-]+[\\-]{5}END[a-z\\s]+PRIVATE\\sKEY|ssh-rsa\\s*[a-z0-9\\/\\.+]{100,}',
+        origin: 'default'
+      },
+      { name: 'remoteConfig.enabled', value: true, origin: 'env_var' },
```

Comment:
Is `env_var` the correct origin value for `remoteConfig.enabled`?

Reply:
Same here,
```
    this._setBoolean(env, 'remoteConfig.enabled', coalesce(
      DD_REMOTE_CONFIGURATION_ENABLED,
      !this._isInServerlessEnvironment()
    ))
```

where:
```
!this._isInServerlessEnvironment()
```

Not in this PR, as here you're only enabling the tests, but I think that we should move the `this._isInServerlessEnvironment()` to `default` values and not to `env`.